lib/scheduler.pm

```diff
@@ -32,6 +32,17 @@ our @EXPORT = qw(load_yaml_schedule get_test_suite_data);
 
 my $test_suite_data;
 my $root_project_dir = dirname(__FILE__) . '/../';
+our @cloudmodules = qw(
```

Comment:
This doesn't belong here, please move it somewhere else not to mix purposes of the modules.

Reply:
i cant think where else it belogs. it has just one purpose which is part of the scheduling. if i move it somewhere else and export it, it would be kind hidden. because, at least at the moment, i have no idea where this should be to make sense.

Reply:
feefbf0